lib/trie/TrieDB.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache2
```

Comment:
This file has been deleted in the base branch

Reply:
Removed the lib/trie folder whole. This is resolved now!